cmd/gaiad/genaccounts.go

```diff
@@ -132,6 +136,17 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 
 			appState[auth.ModuleName] = authGenStateBz
 
+			bankGenState := bank.GetGenesisStateFromAppState(cdc, appState)
```

Comment:
Now in addition to adding the account to the accounts genesis state, we also add the balances to the bank genesis state.